include/ginkgo/core/matrix/dense.hpp

```diff
@@ -539,6 +543,42 @@ class Dense
                     ptr_param<const LinOp> beta,
                     ptr_param<LinOp> row_collection) const;
 
+    /**
+     * Copies this matrix into the given rows of the target matrix.
```

Comment:
I think this needs to be a bit more expressive in what it means to "copy this matrix into the given rows". Could you add an example here.

Reply:
TBH, since this is the brief description, I think it's fine. But the detailed description is mangled up in the description of the parameters, so perhaps moving that into a separate description block is better.